### PR TITLE
Update dependencies

### PR DESCRIPTION
**Pull request by stefpiatek**
Oopf sorry about the whitespace. Turns out we don't get any new travis CI credits anymore and they ran out at the end of last year. Moved CI to github actions

### CODE REVIEW

File: inform-annotations/pom.xml [hunk 1]
```diff
@@ -1,105 +1,68 @@
...
+    <dependency>
+      <groupId>com.google.auto.service</groupId>
+      <artifactId>auto-service</artifactId>
+      <version>${auto-service.version}</version>
+      <scope>provided</scope>
+    </dependency>
+    <dependency>
+      <groupId>org.hibernate</groupId>
+      <artifactId>hibernate-core</artifactId>
+      <version>${hibernate-core.version}</version>
```
Comment (skeating) [inline inform-annotations/pom.xml:32]: defined  ?

File: inform-db/pom.xml [hunk 2]
```diff
@@ -1,136 +1,93 @@
...
-			<artifactId>junit-jupiter-params</artifactId>
-			<version>5.7.0</version>
-			<scope>test</scope>
-		</dependency>
-	</dependencies>
+  <dependencies>
+    <dependency>
+      <groupId>org.hibernate</groupId>
+      <artifactId>hibernate-core</artifactId>
+      <version>${hibernate-core.version}</version>
```
Comment (skeating) [inline inform-db/pom.xml:33]: defined ?

File: inform-db/pom.xml [hunk 3]
```diff
@@ -1,136 +1,93 @@
...
+    </dependency>
+    <dependency>
+      <groupId>org.projectlombok</groupId>
+      <artifactId>lombok</artifactId>
+      <version>${lombok.version}</version>
+    </dependency>
+    <dependency>
+      <groupId>org.junit.jupiter</groupId>
+      <artifactId>junit-jupiter</artifactId>
+      <version>${junit-jupiter.version}</version>
```
Comment (skeating) [inline inform-db/pom.xml:48]: ?

File: inform-db/pom.xml [hunk 4]
```diff
@@ -1,136 +1,93 @@
...
+      <version>${reflections.version}</version>
+      <scope>test</scope>
+    </dependency>
+    <dependency>
+      <groupId>uk.ac.ucl.rits.inform</groupId>
+      <artifactId>inform-annotations</artifactId>
+      <version>${inform-annotations.version}</version>
+    </dependency>
+    <dependency>
+      <groupId>org.junit.jupiter</groupId>
```
Comment (skeating) [inline inform-db/pom.xml:63]: repeat

**Review by skeating — CHANGES_REQUESTED** [review]

File: inform-annotations/pom.xml (hunk 1 above)
Comment (stefpiatek) [reply inform-annotations/pom.xml:32]: sure is, but in the parent pom <repo>/pom.xml

File: inform-db/pom.xml (hunk 4 above)
Comment (stefpiatek) [reply inform-db/pom.xml:63]: sure is, but in the parent pom <repo>/pom.xml

File: inform-db/pom.xml (hunk 3 above)
Comment (stefpiatek) [reply inform-db/pom.xml:48]: sure is, but in the parent pom <repo>/pom.xml

**Review by skeating — APPROVED** [review]

**Pull request merged**